modular_zubbers/modules/jobs/job_types/blacksmith.dm

```diff
@@ -0,0 +1,61 @@
+/datum/job/blacksmith
+	title = JOB_BLACKSMITH
+	description = "Smith wares, Sell them."
+	department_head = list(JOB_HEAD_OF_PERSONNEL)
+	faction = FACTION_STATION
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = SUPERVISOR_HOP
+	config_tag = "BLACKSMITH"
+	exp_granted_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/blacksmith
+	plasmaman_outfit = /datum/outfit/plasmaman/curator //Not sure how this works. Leaving it alone.
```

Comment:
Plasmamen have separate job outfits, as they have to wear something that's more akin to a hardsuit than regular clothing.